packages/kibana/data_stream/background_task_utilization/manifest.yml

```
@@ -0,0 +1,17 @@
+type: metrics
+title: Kibana background task utilization metrics
+release: beta
+dataset: kibana.http_metrics.background_task_utilization
```

```suggestion
dataset: kibana.background_task_utilization
```

I don't think it's needed to add the type into the dataset name. We've added that to the Stack Monitoring legacy streams to not block the usage of that name for when/if new data streams are introduced for the "same" data but natively to the Integration.

---

Updated in https://github.com/elastic/integrations/pull/6169/commits/051981fcf7b8bf87baccf74fd1446c7ded08a9fd